### DISCUSSION TITLE
Organice with Working Copy on iOS

### DISCUSSION THREAD

**Discussion by Kardyne** [200ok-ch/organice#775] [Q&A] [answered]
Hi everyone,

I really like using organice to write orgmode files on the go.
I don't really like the way it handles saving when using a git backend though: there is no control over the commit contents or the message.

I tried using Working Copy on iOS with the integrated WebDAV server to have more control over commits. I was not able to make it work: after connecting I was just greeted by a blank page with no error message.
I couldn't find a way to check the safari web console on iOS.

Did anyone have any luck with this setup? Is there any other way to use organice while choosing what and when to commit?

I am not sure if this is the right place to ask for this.

**Comment by schoettl** [comment, marked as answer]
Hi,

right, there are no commit messages with the git backend. There is a setting "live sync" which automatically saves (commits) the file. If you disable this option, it only saves the file when you press the sync button.

Well, Organice is not a version control system. It just saves files to various backends. If custom commit messages are important, maybe you can use Dropbox as sync backend and commit changes manually. Maybe there is even a Git app that works with Dropbox?
